src/index.html

```diff
@@ -4,10 +4,53 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <div class="product-box">
+
+        <img
+          class="product-image"
+          src="./images/imac.jpeg"
+          alt="product image"
+        >
+
+        <h1>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+        <h3>Product code: 195434</h3>
+
+        <div class="product-stars">
+          <div class="stars-block">
+            <ul class="stars stars--4">
+              <li class="stars__element"></li>
+              <li class="stars__element"></li>
+              <li class="stars__element"></li>
+              <li class="stars__element"></li>
+              <li class="stars__item"></li>
```

Comment:
Need to write correctly classes
```suggestion
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
              <li class="stars__star"></li>
```